frontend/dockerfile/dockerfile_test.go

```diff
@@ -1999,6 +2007,7 @@ ADD %s /dest/
 }
 
 func testDockerfileAddArchive(t *testing.T, sb integration.Sandbox) {
+	skipDockerd(t, sb)
```

Comment:
curious what was the error for this one?

Reply:
```
--- FAIL: TestIntegration (4.32s)                                                                                                                 
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:36763/library/busybox:latest                                     
    run.go:219: copied docker.io/amd64/alpine:latest to local mirror localhost:36763/library/alpine:latest                                       
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:37249/library/busybox:latest                                     
    run.go:219: copied docker.io/amd64/alpine:latest to local mirror localhost:37249/library/alpine:latest                                       
    run.go:219: copied docker.io/amd64/alpine:latest to local mirror localhost:44681/library/alpine:latest                                       
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:44681/library/busybox:latest                                     
    --- FAIL: TestIntegration/TestBuildPushAndValidate/worker=dockerd (4.59s)                                                                    
        require.go:794:                                                                                                                           
                Error Trace:    client_test.go:1580                                                                                               
                                                        run.go:164                                                                                
                Error:          Received unexpected error:                                                                                        
                                rpc error: code = Unknown desc = exporter "image" could not be found                                             
                                failed to solve                                                                                                   
                                github.com/moby/buildkit/client.(*Client).solve.func2                                                            
                                        /go/src/github.com/docker/docker/buildkit/client/solve.go:203                                            
                                golang.org/x/sync/errgroup.(*Group).Go.func1                                                                     
                                        /go/src/github.com/docker/docker/buildkit/vendor/golang.org/x/sync/errgroup/errgroup.go:57               
                                runtime.goexit                                                                                                    
                                        /usr/local/go/src/runtime/asm_amd64.s:1357                                                                
                Test:           TestIntegration/TestBuildPushAndValidate/worker=dockerd                                                           
        oci.go:74: creatingDaemon                                                                                                                 
        oci.go:77: [d558988a16523] waiting for daemon to start[d558988a16523] waiting for daemon to start[d558988a16523] daemon started           
        oci.go:77: [d558988a16523] stopping daemon[d558988a16523] exiting daemon[d558988a16523] daemon stopped                                    
FAIL                                                                                                                                              
FAIL    github.com/moby/buildkit/client 10.541
```

Reply:
```
--- FAIL: TestIntegration (10.95s)                                                                                                                
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:34343/library/busybox:latest                                      
    run.go:219: copied docker.io/docker/dockerfile-copy:v0.1.9@sha256:e8f159d3f00786604b93c675ee2783f8dc194bb565e61ca5788f6a6e9d304061 to local mirror localhost:34343/docker/dockerfile-copy:v0.1.9
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:37649/library/busybox:latest                                      
    run.go:219: copied docker.io/docker/dockerfile-copy:v0.1.9@sha256:e8f159d3f00786604b93c675ee2783f8dc194bb565e61ca5788f6a6e9d304061 to local mirror localhost:37649/docker/dockerfile-copy:v0.1.9
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:38031/library/busybox:latest                                       
    run.go:219: copied docker.io/docker/dockerfile-copy:v0.1.9@sha256:e8f159d3f00786604b93c675ee2783f8dc194bb565e61ca5788f6a6e9d304061 to local mirror localhost:38031/docker/dockerfile-copy:v0.1.9
    run.go:219: copied docker.io/amd64/busybox:latest to local mirror localhost:34943/library/busybox:latest                                      
    run.go:219: copied docker.io/docker/dockerfile-copy:v0.1.9@sha256:e8f159d3f00786604b93c675ee2783f8dc194bb565e61ca5788f6a6e9d304061 to local mirror localhost:34943/docker/dockerfile-copy:v0.1.9
    --- FAIL: TestIntegration/TestDockerfileAddArchive/worker=dockerd/frontend=builtin (1.52s)                                                    
        require.go:794:                                                                                                                           
                Error Trace:    dockerfile_test.go:2048        
                                                        run.go:164                                                                                
                Error:          Received unexpected error:                                                                                        
                                exec: "buildctl": executable file not found in $PATH                                                              
                Test:           TestIntegration/TestDockerfileAddArchive/worker=dockerd/frontend=builtin                                          
        oci.go:74: creatingDaemon                                                                                                                 
        oci.go:77: [d851e2a4f0f71] waiting for daemon to start[d851e2a4f0f71] waiting for daemon to start[d851e2a4f0f71] daemon started
        oci.go:77: [d851e2a4f0f71] stopping daemon[d851e2a4f0f71] exiting daemon[d851e2a4f0f71] daemon stopped
FAIL
```

Reply:
We should make `buildctl` available for these tests.

Reply:
Can we do that in a followup PR since this one has been open for a while? I've got a list of minor followups I can add this to